src/components/patch/Patch.tsx

```diff
@@ -34,34 +35,35 @@ export class Patch extends React.Component<Props, State> {
       datetime: datetime,
       project: this.props.Patch.Version.identifier,
       author: this.props.Patch.Version.author,
+      builds: this.props.Patch.Builds
     };
   }
 
   public render() {
 
+    const Variants = () => (
+      <Grid container={true} spacing={24}>
+        <Grid item={true} xs={12}>
+          <Typography>
+            Applied at {this.state.datetime.format("D MMM YYYY, h:mm a")} on {this.state.project}
```

Comment:
The user's browser locale is not something I had thought about before but I definitely agree. I changed the places that use a datetime to format based on locale. I also swapped out "applied" for "created" (I didn't like that wording either, I just kept it because I wasn't sure if the word applied was an intentional choice with strong reasoning behind it)